base/src/main/java/com/smartdevicelink/transport/SiphonServer.java

```diff
@@ -234,14 +234,11 @@ private static Boolean sendSiphonData(SiphonDataType direction, byte[] msgBytes,
 		byte siphonDataTypeIndicator = 0x00;
 		
 		long currentDateTime = System.currentTimeMillis();
-		Integer deltaTimeMills = null;
+		Integer deltaTimeMills;
 			
 		deltaTimeMills = (int)(currentDateTime - m_startTimeStamp);
 		
 		switch(direction) {
-			case fromSdl:
-				siphonDataTypeIndicator = 0x00;
-				break;

```

Comment:
Why was this removed?

Reply:
Because it is covered in the default entry. But I will add it back if you think it makes it less readable.

Reply:
I think it is fine to leave it out